### PR TITLE
Add zipcodes definitions

### PR DESCRIPTION
**Pull request by headdetect**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Review by plantain-00 — CHANGES_REQUESTED** [review]
I don't think the `README.md` is needed here, it will be generated automatically when published to npm.

**Comment by headdetect** [comment]
@plantain-00 Removed the README.

**Review by plantain-00 — APPROVED** [review]

File: types/zipcodes/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+export interface ZipCode {
+  zip: string;
+  latitude: number;
+  longitude: number;
+  city: string;
+  state: string;
+  country: string;
+}
+
+export namespace ZipCodes {
```
Comment (mhegazy) [inline types/zipcodes/index.d.ts]: i do not think this is correct..:
```sh
> var z = require("zipcodes");
undefined
> z.ZipCodes
undefined
```

I think you are missing `export = ZipCodes;`  in this file.

**Review by mhegazy — CHANGES_REQUESTED** [review]
I do not think these type definitions match that of the actual package.

`dts-gen` seems to produce the correct file. consider runnging:

```sh
npm install -g dts-gen

dts-gen -m zipcodes -dt zipcodes
```

File: types/zipcodes/index.d.ts (hunk 1 above)
Comment (headdetect) [reply types/zipcodes/index.d.ts]: Wouldn't that still be the same as `export namespace ZipCodes`, no?

**Pull request merged**